### PR TITLE
Create HarvardMed_Aganj.yaml

### PR DESCRIPTION
**Pull request by ChristinaLK**

### CODE REVIEW

**Comment by mwestphall** [comment]
Hi @opensciencegrid/project-office , are we good to add "Harvard Medical School" as an institution to the institutions database?

**Comment by matyasselmeci** [comment]
Harvard Medical School should already be in there: as `https://osg-htc.org/iid/n1kbnzl7kyiv`. We have other projects with that institution.

**Comment by matyasselmeci** [comment]
for example: https://github.com/opensciencegrid/topology/blob/master/projects/NGNDA.yaml

**Comment by mwestphall** [comment]
@matyasselmeci that might be indicative of a gap in our validation logic, the "n1kbnzl7kyiv" iid corresponds to "Harvard University" in the institutions DB. Do we want/need a separate institution for the medical school?

**Comment by matyasselmeci** [comment]
Good question. Harvard Medical School does not show up in ror.org: https://ror.org/search?query=Harvard+Medical+School

**Comment by matyasselmeci** [comment]
I take that back. The ROR registration has been withdrawn: https://ror.org/03wevmz92 with the comment "This organization is succeeded by Harvard University (https://ror.org/03vek6s52)."

@opensciencegrid/project-office in the past we have treated Harvard Medical School as a separate organization from Harvard University -- do we still want to do that?

**Comment by jstathas** [comment]
Especially given their actions in ROR, I believe the correct answer is that we should use: Harvard University. Confirmed just now by BB. The rule is the most specific institution within the ROR list.

Janet

From: Matyas Selmeci ***@***.***>
Date: Thursday, October 3, 2024 at 11:55 AM
To: opensciencegrid/topology ***@***.***>
Cc: Stathas, Janet ***@***.***>, Team mention ***@***.***>
Subject: Re: [opensciencegrid/topology] Create HarvardMed_Aganj.yaml (PR #4214)

I take that back. The ROR registration has been withdrawn: https://ror.org/03wevmz92 with the comment "This organization is succeeded by Harvard University (https://ror.org/03vek6s52)."

@opensciencegrid/project-office<https://github.com/orgs/opensciencegrid/teams/project-office> in the past we have treated Harvard Medical School as a separate organization from Harvard University -- do we still want to do that?

File: projects/HarvardMed_Aganj.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  \ involved in aging, Alzheimer\u2019s disease, and Parkinson\u2019s disease, and\
+  \ uncover new imaging biomarkers to improve diagnosis and treatment. Normal aging,\
+  \ as well as occurrence of neurodegenerative diseases, can be better understood\
+  \ by mapping complex structural and functional brain networks through which information\
+  \ flows, and via the discovery of important biomarkers that shed light on aging\
+  \ and neurodegenerative disease."
+FieldOfScience: Biomedical research
+FieldOfScienceID: '14.0501'
+InstitutionID: Unknown
+Organization: Harvard Medical School
```
Comment (matyasselmeci) [inline projects/HarvardMed_Aganj.yaml:13]:
```suggestion
InstitutionID: https://osg-htc.org/iid/n1kbnzl7kyiv
Organization: Harvard University
```

**Comment by matyasselmeci** [comment]
Thanks! Two more questions:

1. Should we (if we can) change the project name from `HarvardMed_Aganj` to `Harvard_Aganj`?
2. Should we (separately of course) change the Organizations of the other projects that use "Harvard Medical School"?

**Comment by jstathas** [comment]
1.
Yes
  2.
Yes


Janet Stathas (she/her)

Project Manager

Morgridge Institute for Research

PATh<https://path-cc.io/>

Email: ***@***.***

Mobile: 608.345.9605

________________________________

**Review by mwestphall — APPROVED** [review]

**Pull request merged**